cnf-tests/testsuites/e2esuite/security/tuning.go

```diff
@@ -94,4 +94,38 @@ var _ = Describe("tuningcni", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 	})
+
+	Context("tuningcni over bond", func() {
+		It("should be able to create pod with sysctls over bond",
```

Comment:
Can't this test be integrated in the test being added in https://github.com/openshift-kni/cnf-features-deploy/pull/1129 ?
The tests execution time will benefit from this

Reply:
Absolutely. I believe no additional changes are needed besides dropping this one
The tests will be a bit less granular, but the time saving is probably worth it.